docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,92 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Definitions
```

Comment:
I believe this needs a rebase to avoid showing the merged commit to the REQUIREMENTS.md.

Reply:
rebased!